pkg/kubelet/stats/cadvisor_stats_provider.go

```diff
@@ -91,9 +91,8 @@ func (p *cadvisorStatsProvider) ListPodStats() ([]statsapi.PodStats, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to get container info from cadvisor: %v", err)
 	}
-	// removeTerminatedContainerInfo will also remove pod level cgroups, so save the infos into allInfos first
-	allInfos := infos
-	infos = removeTerminatedContainerInfo(infos)
+
```

Comment:
I think these could use more informative names now that we've lost this comment:
`filteredInfos, allInfos` perhaps?